karma.conf.js

```diff
@@ -29,7 +29,7 @@ var karmaConfig = {
   port: 9876,
   colors: true,
   autoWatch: false,
-  browsers: ['ChromeHeadless', 'FirefoxHeadless'],
```

Comment:
Not sure that we can deliver stuff that works only in Firefox :))

Reply:
Why not? Who's still using Google products anyway? :)). Joking aside, I just wanted to get this passed the browser tests, so you guys don't have an excuse to not start reviewing. I guess this needs to be addressed in the underlying library
https://github.com/antouhou/bls-signatures/blob/feature/js_build/js-bindings/tests/karma.conf.js#L21https://github.com/antouhou/bls-signatures/blob/feature/js_build/js-bindings/tests/karma.conf.js#L21
@antouhou any ideas? In any case we should probably recompile it and move to @dashevo?
In the meantime I will play around with the native javascript implementation of bls12-381
https://github.com/paulmillr/noble-bls12-381
Maybe I can tweak the curve to output exactly the same as the chia implementation. If that were possible we can just use native javascript in all browsers that don't support wasm (Chrome supports wasm btw, so the fact that it doesn't pass the tests must stem from another problem).

Reply:
Yeah, we can call asynchronous functions which use chia bls in order to solve that problem for Chrome, or use that lib what you suggest. We need to make some research on it.